insights/parsers/satellite_settings.py

```diff
@@ -0,0 +1,77 @@
+"""
+SatelliteSettings - command ``psql -d foreman -c 'select name, value, "default" from settings where name in (\'destroy_vm_on_host_delete\', \'unregister_delete_host\')'``
+==========================================================================================================================================================================
+
+The SatelliteSettings parser reads the output of
+``psql -d foreman -c 'select name, value, "default" from settings where name in (\'destroy_vm_on_host_delete\', \'unregister_delete_host\')'``.
+
+Sample output of ``psql -d foreman -c 'select name, value, "default" from settings where name in (\'destroy_vm_on_host_delete\', \'unregister_delete_host\')'``::
+
+            name            |  value   |  default
+    ---------------------------+----------+-----------
+    unregister_delete_host    | --- true+| --- false+
+                            | ...      | ...
+    destroy_vm_on_host_delete |          | --- true +
+                            |          | ...
+    (2 rows)
+
+
+Examples::
+
+    >>> type(settings)
+    <class 'insights.parsers.satellite_settings.SatelliteSettings'>
+    >>> 'unregister_delete_host' in settings
+    True
+    >>> settings['unregister_delete_host']
+    True
+    >>> len(settings)
+    2
+    >>> settings['destroy_vm_on_host_delete']
+    True
+"""
+
+import os
+import yaml
+
+from insights import parser, CommandParser
+from insights.specs import Specs
+from insights.parsers import SkipException, ParseException
+
+
+@parser(Specs.satellite_settings)
+class SatelliteSettings(CommandParser, dict):
```

Comment:
`list` would be better to make the parser more compatible with the SQL passed to the spec.

Just as mentioned this [comment](https://github.com/RedHatInsights/insights-core/pull/2934#issuecomment-773917878). It's possible to add more columns in the future for other requirements.

Reply:
@xiangce, sure, thanks. I'll update the parser to make it more extendable based on your suggested code.